### DISCUSSION TITLE
How do a view status screen after stopping the afl-fuzz?

### DISCUSSION THREAD

**Discussion by lyonnhk** [AFLplusplus/AFLplusplus#1356] [Q&A]
I have fuzzed some targets and i had to stop my fuzzer to do a system update. Now when i continue fuzzing, the status screen only shows new crash values etc. Is it possible to view the status screen values from my previous fuzz attempts? i understand that the values are stored in fuzzer_stats however how do i display them in the status screen format. Thank you.

![image](https://user-images.githubusercontent.com/57974924/159050288-e513e39e-0e79-4aec-9517-8299999c5a37.png)

**Comment by domenukk** [comment]
https://github.com/AFLplusplus/AFLplusplus/blob/10dae419d6e3ebc38f53840c5abfe98e9c901217/src/afl-fuzz-stats.c#L178

I would assume this line already restores the amount of saved crashes on resume 🤔

**Reply by lyonnhk** [reply]
Yeah thats what i assume too however my crashes saved resets to 0 when i continue the program..

**Comment by vanhauser-thc** [comment]
could it be that you are not running a current version? (4.0+)

**Reply by lyonnhk** [reply]
i am currently using the latest version of AFL++, had just cloned it on my pc.
